zorg/buildbot/builders/ClangBuilder.py

```diff
@@ -284,10 +285,12 @@ def _getClangCMakeBuildFactory(
     # and the test-suite separately. Le's do this first,
     # so we wouldn't poison got_revision property.
     if runTestSuite or checkout_test_suite:
-        f.addGetSourcecodeForProject(
-            project='lnt',
-            src_dir='test/lnt',
-            alwaysUseLatest=True)
+        if testWithLNT:
+            f.addGetSourcecodeForProject(
+                project='lnt',
+                src_dir='test/lnt',
```

Comment:
This is off topic really, but it has always annoyed me that repos get checked out to something other than the default name. It makes reproducing locally just a little bit more annoying.

There are others so I'll go through them at some point, no point changing this one now.

Reply:
I agree with your point but changing directory name should be fairly silent.